deep-learning/_posts/2017-08-03-video_propagation_networks.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: review
+title:  "Video Propagation Networks"
+tags:   deep-learning CNN
+author: Clément Zotti
+pdf:   "https://varunjampani.github.io/papers/jampani17_VPN.pdf"
+
+cite:
+
+  authors: "V. Jampani, R. Gadde and P.V. Gehler"
+  title:   "Video Propagation Networks"
+  venue:   "CVPR 2017"
+---
+
+Paper :
+   - [project page](https://varunjampani.github.io/vpn/)
+   - [code](https://github.com/varunjampani/video_prop_networks)
+   - [suplementary material](https://varunjampani.github.io/papers/jampani17_VPN_supp.pdf)
+
+
+The goal of this papers is to take a video and a task image as input and apply the task image on the entire video, this task could be pixel segmentation, pixel colorization, etc.
+
+## Models
+
+To achieve this they use two kinds of network, a [Bilateral network](https://arxiv.org/pdf/1412.6618.pdf){:target="_blank"}, and a Spatial network.
+
+The bilinear features used are defined by a 6 dimensional feature vector $$ F^i = (x, y, Y, Cb, Cr, t)^T $$. Where $$x$$ and $$y$$ are the pixel position in the image, $$Y$$, $$Cb$$, and $$Cr$$ the luma, blue-difference and red-difference chroma components, and $$t$$ the time of the frame in the video.
+
+The bilateral network performs video-adaptive spatiotemporal dense filtering that is given to the spatial network (regular CNN) used to produce the new segmentation from the previous frame segmentation as shown in Fig.3.
```

Comment:
what is guidance in Fig.3?

Reply:
The Fig.3 is there to show the architecture and the connection between the bilateral network and the spatial network as well as the input of the bilateral network.